helm/templates/certificate_webhook-conf.yaml

```diff
@@ -33,6 +33,11 @@ metadata:
 webhooks:
   - name: {{ .Chart.Name }}-svc.{{ .Release.Namespace }}.svc
     failurePolicy: Ignore
+    {{- if .Values.deployment.reinvocationPolicy }}
+    reinvocationPolicy: {{ .Values.deployment.reinvocationPolicy }}
```

Comment:
I think this could be simplified to `{{ .Values.deployment.reinvocationPolicy | default "Never" }}`

Reply:
I agree. I just want to follow the same style that are being used for `failurePolicy`

Reply:
Ah, makes sense. I'd vote for refactoring the failurePolicy expression unless @phbelitz has a different opinion

Reply:
I agree with @Starkteetje and we should go with using the `default` template function. We'll probably rework the helm chart at some point and swicth styles for `failurePolicy` and the likes as well.

Reply:
ok, let me update it 
Renault ZOE INTENS 41kWh zzgl. Batterie Miete KLIMAAUTO
7 / 29 Vergrößern

Reply:
ok, let me update it

Reply:
updated for both `failurePolicy` and `reinvocationPolicy` :)